navis/interfaces/h01.py

```diff
@@ -0,0 +1,40 @@
+from functools import lru_cache
+from . import cave_utils
+
+@lru_cache(None)
+def get_cave_client():
+    """Get caveclient for H01 dataset.
+    """
+    datastack = "h01_c3_flat"
```

Comment:
I'd recommend putting these variables at module-level (all capital by convention):

```python
from . import CAVE_UTILS

DATASTACK = "h01_c3_flat"
SERVER_ADDRESS = "https:///...."
```

That way, users can edit this during runtime if they have to.